config/default.yml

```diff
@@ -2557,6 +2557,7 @@ Naming/InclusiveLanguage:
         - denylist
         - block
     slave:
+      WholeWord: true
```

Comment:
This can also be named "PreciseMatch" or something like this. Sounds a bit more descriptive to me.

Reply:
Or "ExactMatch". I think we use this somewhere else in the config. It'd be best to check, so we stay consistent with ourselves.

Reply:
We have `ExactNameMatch` in `Style/TrivialAccessors` but I don't think it exactly fits here. `PreciseMatch` would be ok with me but also I wanted to indicate that `foo_slave` *would* be caught, because it's a different "word". If you like `PreciseMatch` I'm happy to change it.

Reply:
Fair point. Let's stick with `WholeWord` then. Naming is hard.